### PR TITLE
Release changelog for 2.8

### PR DESCRIPTION
**Pull request by sergiusens**
Signed-off-by: Sergio Schvezov <sergio.schvezov@ubuntu.com>

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,44 @@
...
+  * Copy plugin: Follow symlinks if outside of snap. (LP: #1532515)
+  * Lifecycle: Check to see if prerequisite is already built. (#431)
+    (LP: #1537790)
+  * Clean command: Don't automatically clean reverse dependencies. (#436)
+    (LP: #1568004)
+  * Library crawling: Make ldd failure non-fatal. (#440) (LP: #1569280)
+  * Extract step state tracking into its own module. (#438) (LP: #1477904)
+  * Automatically re-strip if dirty. (#442) (LP: #1477904)
+  * Automatically re-stage if dirty. (#443) (LP: #1477904)
+  * Detect dirty build step. (#447) (LP:  #1477904)
```
Comment (kyrofa) [inline debian/changelog]: Aw man, did I screw that up? Looks like there's one too many spaces in the bug reference.

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,44 @@
...
+  * Support .git extension detection (LP: #1566153)
+
+  [ Kyle Fazzari ]
+  * Copy plugin: Follow symlinks if outside of snap. (LP: #1532515)
+  * Lifecycle: Check to see if prerequisite is already built. (#431)
+    (LP: #1537790)
+  * Clean command: Don't automatically clean reverse dependencies. (#436)
+    (LP: #1568004)
+  * Library crawling: Make ldd failure non-fatal. (#440) (LP: #1569280)
+  * Extract step state tracking into its own module. (#438) (LP: #1477904)
```
Comment (kyrofa) [inline debian/changelog]: You can probably get rid of this one-- it means nothing to our users.

File: debian/changelog [hunk 3]
```diff
@@ -1,3 +1,44 @@
...
+    (LP: #1536691, #1561068)
+  * Remove global state attributes from snapcraft.common (#434) (LP: #1551849)
+  * Remove support and mentions of the config hook. (#437) (LP: #1568826)
+  * Remove `old-security`. Use proper interfaces. (#441) (LP: #1569452)
+  * Make meta a class in snapcraft.internal.meta (#444) (LP: #1551849)
+  * Move common directories to ProjectOptions (#445) (LP: #1551849)
+
+  [ Leo Arias ]
+  * Temporary skip the webcam example while we debug the pip issue.
+    (LP: #1567452)
```
Comment (kyrofa) [inline debian/changelog]: This has been reverted though, right? We can likely remove it either way.

**Comment by elopio** [comment]
I've verified the bugs, did some exploratory and the examples build, which is as far as we can go today, so :+1: 

Standing ovation for the removal of the globals in common. :clap:

**Pull request merged**